src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -143,14 +146,121 @@ public Integer getTotalUnReviewedDars(AuthUser authUser) {
         return darManage;
     }
 
+    private int getSortDateInt(Document document) {
+        Long time = document.getLong(DarConstants.SORT_DATE);
+        if (time == null) {
+            time = new Date().getTime();
+        }
+        return time.intValue();
+    }
+
     private Map<String, Election> createAccessRequestElectionMap(List<Election> elections) {
         Map<String, Election> electionMap = new HashMap<>();
         elections.forEach(election -> electionMap.put(election.getReferenceId(), election));
         return electionMap;
     }
 
+    /**
+     * Convenience method during transition away from `Document` and to `DataAccessRequest`
+     */
```

Comment:
Add todos here so we know which ones need to be deleted in subsequent cleanup PRs.